csrc/id_model/indexing.cpp

```diff
@@ -369,7 +370,26 @@ void TensorIndexer::buildLoopIndexMap() {
           shouldUseZeroIndex(loop_group) || isParallelTypeDeviceDim(ptype)) {
         loop_index = fusion->zeroVal();
       } else {
-        loop_index = IrBuilder::create<Val>(DataType::Index);
+        // Until the transition to the IdModel-based indexing is
```

Comment:
This is necessary for now since loops are generated using the ComputeAtMap indices.